src/main.rs

```diff
@@ -41,6 +41,37 @@ struct Opts {
     /// Reduce detail of internal logging. Repeat to reduce further. Overrides `--verbose`.
     #[structopt(short, long, parse(from_occurrences))]
     quiet: u8,
+
+    /// Control when ANSI terminal formatting is used.
+    ///
+    /// By default `vector` will try and detect if `stdout` is a terminal, if it is
+    /// ANSI will be enabled. Otherwise it will be disabled. By providing this flag with
+    /// the `--color ansi` option will always enable ANSI terminal formatting.
+    ///
+    /// Options: `ansi` or `none`
+    #[structopt(long)]
+    color: Option<Color>,
+}
+
+#[derive(Debug, Clone, PartialEq)]
+enum Color {
+    Ansi,
+    None,
```

Comment:
I would just go with `auto`, `always`, `never` for these. I feel like that matches most tools I've seen.